spec/support/mailer.rb

```diff
@@ -0,0 +1,129 @@
+# @see https://thoughtbot.com/blog/building-custom-rspec-matchers-with-regular-objects
+module Spec
+  module Support
+    module Mailer
+      def be_to(email_addresses)
+        BeToMatcher.new(email_addresses)
+      end
+
+      def have_subject(subject)
+        HaveSubjectMatcher.new(subject)
+      end
```

Comment:
I personally don't see a ton of value in these two matchers -- it seems like testing the contents of `mail.to` and `mail.subject` is simple enough that we don't need to wrap them in helpers. 🤷🏼‍♀️

Reply:
Yea, there's not really a *great* reason for them other than "I like them and think they're neat" which is something I need more of in my life right now.

In particular, I like how they make tests feel, because you can do cool stuff like `it { is_expected.to be_to('person@example.com').exclusively }` on the subject of the example group. I was pretty averse to this kind of custom matchers back when it took their god-awful DSL to define them; but now that they're POROs I'm pretty excited to explore how expressive we can make our tests.